enterprise/views.py

```diff
@@ -416,3 +419,121 @@ def post(self, request):
             # We're handing consent only for a particular course
             return self.post_course_specific_consent(request, specific_course, consent_provided)
         return self.post_account_consent(request, consent_provided)
+
+
+class LandingPageView(View):
+    """
+    Enterprise landing page view.
+
+    This view will display the course mode selection with related enterprise
+    information.
+    """
+
+    enterprise_welcome_text = _(
+        "{strong_start}{enterprise_customer_name}{strong_end} has partnered with "
+        "{strong_start}{platform_name}{strong_end} to offer you high-quality learning "
+        "opportunities from the world's best universities."
+    )
+    confirmation_text = _("Confirm your course")
+    starts_at_text = _("Starts")
```

Comment:
I think it would be better to put all constant string in a dict, dictionary can be defined at the top of the file. Thoughts ?